### DISCUSSION TITLE
Objectives for May

### DISCUSSION THREAD

**Discussion by udayRage** [UdayLab/PAMI#401] [General]
We need to optimize the following algorithms:

1. FP-MAX
2. ECLAT Diffset
3. CoMine and CoMine++  (minor modification)
4. RSFP-growth
5. CFP-growth and CFP-growth++ (minor modification)
6. CMine and CMine++ (minor modification)

**Comment by vanithakattumuri** [comment]
# The optimised algorithms and updated documentation:
1. FAE
2. ECLAT Diffset
3. CoMine
4. PFPGrowth
5. PFECLAT

# In Process:
1. FP-MAX
2. CoMine++
3. RSFP-growth
7. CFP-growth and CFP-growth++
8. CMine and CMine++
